src/transaction-flow/input/ProfileEditor/ProfileEditor-flow.tsx

```diff
@@ -241,6 +241,17 @@ const ProfileEditor = ({ data = {}, transactions = [], dispatch, onDismiss }: Pr
 
   if (isLoading || resolverStatus.isLoading || !isRecordsUpdated) return <TransactionLoader />
 
+  const handleCancel = () => {
```

Comment:
handleCancel and handleConfirm are only used in the avatar component I think. Can change the name to be specific to the avatar component e.g. `handleAvatarConfirm` or can just use an inline function